src/components/Tool/Appeal/Appeals.tsx

```diff
@@ -33,7 +33,7 @@ const Appeals = (): ReactElement => {
         <Box display="flex" justifyContent="center" mt={10}>
           <LoadingIndicator color="primary" size={40} />
         </Box>
-      ) : data?.primaryAppeal ? (
+      ) : data?.primaryAppeal && data.primaryAppeal.nodes.length > 0 ? (
```

Comment:
I had to add this back because just having 

`data?.primaryAppeal`

Will always return true even if the list is empty so it would crash since there's no .nodes[0]